src/content/en/fundamentals/design-and-ui/input/touch/index.md

```diff
@@ -557,7 +506,7 @@ application:
   </tbody>
 </table>
```

Comment:
Maybe somewhere in here it's worth talking about / pointing to some feature detection/event binding example a la https://w3c.github.io/pointerevents/#examples (which omits the legacy MS prefixed versions, but otherwise gives a rough reference for transitional handling of Pointer/Touch/Mouse and a reminder about Keyboard again)